gap/Functors.gi

```diff
@@ -88,9 +88,9 @@ InstallMethod( ConvertToCellInCategoryOfQuiverRepresentations,
     
     k := LeftActingDomain( A );
     
-    dims := List( ValuesOnAllObjects( F ), Dimension );
+    dims := List( AsList( ValuesOnAllObjects( F ) ), Dimension );
```

Comment:
Shouldn't this be `ListOfValues` instead of `AsList`?

Reply:
_AsList_ worked in this case because it delegates to 
```
function ( list )
    return ConstantTimeAccessList( Enumerator( list ) );
end

```
However, I have should used _ListOfValues_.